src/components/SummaryPageCard/SummaryPageCard.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { ReactElement } from "react";
+import { SummaryPageCardProps } from "./SummaryPageCardTypes";
+import Button from "../Button";
+import {
+	BottomActionWrapper,
+	ContentWrapper,
+	RecentActivityWrapper,
+	TitleWrapper,
+	TitleBar,
+} from "./SummaryPageCard.styled";
+
+const RecentActity = (props: SummaryPageCardProps): ReactElement => {
```

Comment:
This is called RecentActivity and is being exported like that but the file's called SummaryPageCard, and from the props I'm guessing this should be the Card component